libweasyl/libweasyl/models/tables.py

```diff
@@ -664,6 +664,21 @@ def default_fkey(*args, **kwargs):
 Index('ind_searchmapsubmit_targetid', searchmapsubmit.c.targetid)
 
 
+searchmapartist = Table(
+    'searchmapartist', metadata,
+    Column('tagid', Integer(), primary_key=True, nullable=False),
+    Column('targetid', Integer(), primary_key=True, nullable=False),
+    Column('settings', CharSettingsColumn({
```

Comment:
It turns out that searchtag.associate assumes there will be a settings column (all other searchmap tables use it to just store whether it's an artist-set tag or not), so in the interest of reducing complexity I put them in separate tables that each have an unused settings column. If you'd prefer, I could drop the settings and set up a special case for these tags instead but I assumed that wouldn't be ideal.

The other option, I guess, would be to refactor ALL the searchmaps but I considered that beyond the scope of this change.